proto/api/v1/payment.proto

```diff
@@ -203,6 +204,7 @@ message PaymentServiceHasPaymentMethodRequest {
 message PaymentServiceHasPaymentMethodResponse {
   bool exists = 1;
   bool coupon_left = 2;
+  bool positive_balance = 3;
```

Comment:
why positive balance instead of balance ?

Reply:
This method just check if the orga is allowed to do something. For example for users who are not allowed to see the billing but can edit.
So it is just returns `true` if the customer has positive balance, but not the value itself.

Reply:
OK understood